### PR TITLE
Rework the persons page

### PR DESCRIPTION
**Pull request by MrTimscampi**
~~**Note: #983 needs to be merged first, since this contains these commits**~~ It's been merged, so this is good to go.

Reworks the person pages to be in the same style as the artist pages in #983 

image

Also fixes the date format for birth and death dates:
image

### CODE REVIEW

**Review by ferferga — COMMENTED** [review]
Mostly looks good, but I'm not sure about this:

image

The born/death details look misaligned (even if they aren't) just because it's missing a header.

What also about adding this to the left top side, just like "You may also like"?

**Comment by neopc10** [comment]
Stylistically the birthplace/date information could be simplified. A cue from Wikipedia for instance:
image

**Comment by MrTimscampi** [comment]
Changed a bit how the info looks, to be closer to Wikipedia.

image

File: client/locales/en-US.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "artist": "Artist",
   "artists": "Artists",
   "audio": "Audio",
+  "backgroundInformation": "Background information",
```
Comment (ferferga) [inline client/locales/en-US.json:14]: I'm not sure about the wording of this, it's true that in the Wikipedia screenshot posted it appears as a "Background information", but I'm not able to find any other articles about actors that have that wording. There's simply no "Background" in that information, it's just "usual" information you would expect from a person, if that makes sense.

I see "Details" or "Personal information" more fitting.

**Comment by neopc10** [comment]
The information tab is awkward. I'm assuming that the huge photo (which isn't information) is moving into that greyed out photos tab so I'll leave that alone. The two subsections/headers are  biography which essentially means "life information," and personal information. I get it, there's information in the person details information tab.

I would do away with both headers, just like there isn't a header in the movies and shows tabs telling me that they're movies and shows.

image


image

File: client/pages/person/_itemId/index.vue [hunk 2]
```diff
@@ -98,22 +98,11 @@
...
                   <v-col>
-                    <v-img
-                      cover
-                      aspect-ratio="1.7778"
-                      :src="personBackdrop.url"
-                    />
                     <v-row>
                       <v-col cols="12" md="7">
-                        <h2 class="text-h6 mt-2">
-                          <span>{{ $t('biography') }}</span>
```
Comment (ferferga) [inline client/pages/person/_itemId/index.vue]: Are those keys removed?

**Comment by neopc10** [comment]
Is it worth combining the shows and movies tabs back into a unified filmography tab?

**Review by ferferga — APPROVED** [review]

**Comment by ferferga** [comment]
@jellyfin-bot rebase

**Comment by ThibaultNocchi** [comment]
The data paragraph doesn't seem to be top aligned with the biography?
print_screen_2021-05-30-14-44-03

File: client/locales/en-US.json [hunk 3]
```diff
@@ -102,10 +102,15 @@
...
     "crew": "Crew",
     "person": {
-      "birth": "Birth",
+      "birth": "Born",
       "birthPlace": "Birth place",
-      "death": "Death",
-      "person": "Person"
+      "books": "Books",
+      "death": "Died",
+      "information": "information",
```
Comment (ThibaultNocchi) [inline client/locales/en-US.json:109]: Don't we capitalize words?

**Review by ThibaultNocchi — APPROVED** [review]

**Pull request merged**